File: 백준/Bronze/23320. 홍익 절대평가/홍익 절대평가.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int[] As = new int[N];
        for (int i = 0; i < N; i++) {
            As[i] = sc.nextInt();
        }
        int X = sc.nextInt();
        int Y = sc.nextInt();
        int tmp = 0;
        for (int a : As) {
            if (a >= Y) {
                tmp++;
            }
        }
        System.out.println((N * X / 100) + " " + tmp);
    }
}

File: 백준/Bronze/23320. 홍익 절대평가/홍익 절대평가.c
#include <stdio.h>

int main() {
    int N, X, Y, tmp = 0;
    scanf("%d", &N);
    int As[N];
    for (int i = 0; i < N; i++) {
        scanf("%d", &As[i]);
    }
    scanf("%d %d", &X, &Y);
    for (int i = 0; i < N; i++) {
        if (As[i] >= Y) {
            tmp++;
        }
    }
    printf("%d %d\n", (N * X) / 100, tmp);
    return 0;
}